src/types.ts

```diff
@@ -14,3 +14,7 @@ export interface IRow {
 export interface IInteractiveState {
   rows: IRow[];
 }
+
+export const Versions = [ "forces-unknown", "earth", "venus", undefined ] as const;
+export type Version = typeof Versions[number];
```

Comment:
Nice, I've never used this kind of typing. I'd usually say: 
`type Version = "forces-unknown" | "earth" | "venus" | undefined;` 
But your way lets you use it in the runtime in `getVersion()` function 👍

Reply:
Credit goes to Kirk.